Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -777,22 +777,16 @@ private bool IsProjectGlobalDeclaration(Declaration candidateDeclaration, Declar
         {
             return candidateDeclaration.ProjectName == scopingDeclaration.ProjectName
                 && !(candidateDeclaration.ParentScopeDeclaration is ClassModuleDeclaration)
-                && (IsExplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration)
-                || IsImplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration));
+                && (IsPublicInOtherModule(candidateDeclaration, scopingDeclaration));
         }
 
-        private bool IsExplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
+        private bool IsPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
         {
             return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
-                    && candidateDeclaration.Accessibility == Accessibility.Public;
-        }
-
-        private bool IsImplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
-        {
-            return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
-                        && candidateDeclaration.Accessibility == Accessibility.Implicit
-                        && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)
-                        && !candidateDeclaration.IdentifierName.StartsWith("Option ");
+                        && (candidateDeclaration.Accessibility == Accessibility.Public
+                            || (candidateDeclaration.Accessibility == Accessibility.Implicit)
+                                && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)
```

Comment:
This looks like a much better 'wheel' than the one I just reinvented. :/
How would you like me to proceed?  Seems like the options are:
1.  Modify DeclarationFinder to use AccessibilityCheck or,
2. Have the (currently two) clients of the DeclarationFinder call AccessibilityCheck directly. 
3. ?